<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video Converter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0" colspan="5">
     <widget class="QPushButton" name="pushButton_saveChange">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Save changes</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="5">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>60</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="8" column="0" colspan="5">
     <widget class="QPushButton" name="pushButton_convert">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Convert</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="5">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string>Audio Codec</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QComboBox" name="comboBox_audio">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>copy</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>aac -ac 2</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" colspan="5">
     <widget class="QListWidget" name="listWidget_files"/>
    </item>
    <item row="3" column="0" colspan="5">
     <widget class="QPlainTextEdit" name="plainTextEdit_info">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="5">
     <widget class="QPushButton" name="pushButton_info">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Info</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Video Codec</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QComboBox" name="comboBox_video">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>copy</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>h264_nvenc -vf format=yuv420p -b:v 6M</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="5">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Video Index</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="spinBox_video">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Subs Index</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QSpinBox" name="spinBox_subs">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="value">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Audio Index</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSpinBox" name="spinBox_audio">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="9" column="0" colspan="5">
     <widget class="QListWidget" name="listWidget_commands"/>
    </item>
    <item row="6" column="0" colspan="5">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>838</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_info</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>generateInfo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>563</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_convert</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>launchCommands()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>531</x>
     <y>558</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget_files</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>removeFile(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget_commands</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>removeCommand(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>759</x>
     <y>635</y>
    </hint>
    <hint type="destinationlabel">
     <x>826</x>
     <y>554</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_saveChange</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>499</y>
    </hint>
    <hint type="destinationlabel">
     <x>558</x>
     <y>478</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget_files</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateWidgetCommands()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>generateInfo()</slot>
  <slot>launchCommands()</slot>
  <slot>removeFile(QListWidgetItem*)</slot>
  <slot>removeCommand(QListWidgetItem*)</slot>
  <slot>saveChanges()</slot>
  <slot>updateWidgetCommands()</slot>
 </slots>
</ui>
